File: be.kendrickheller.com/core/src/main/java/com/smartcard/common/EnumFileType.java
package com.smartcard.common;

public enum EnumFileType {
	Image(1,"images"),
	Document(2,"documents"),
	Audio(3,"audios"),
	Video(4,"videos"),
	Other(5,"other");
	
	private int value;
	private String subPath;
	private EnumFileType(int value, String subPath) {
        this.value = value;
        this.subPath = subPath;
    }
	public int getValue() {
		return value;
	}
	public String getSubPath() {
		return subPath;
	}
	
	public static EnumFileType fromId(int id) {
        for (EnumFileType type : values()) {
            if (type.getValue() == id) {
                return type;
            }
        }
        return null;
    }
}

File: be.kendrickheller.com/apps/api/src/common/EnumFileType.ts
export enum EnumFileType {
    Image = 1,
    Document = 2,
    Audio = 3,
    Video = 4,
    Other = 5
}

export const EnumFileTypeMap: Record<number, string> = {
    1: 'images',
    2: 'documents',
    3: 'audios',
    4: 'videos',
    5: 'other'
};
